src/utils/sound_utils.py

```diff
@@ -1,7 +1,27 @@
 from src.conf import config
-
+import math
 
 class SoundUtils:
     def __init__(self):
-        # tak się dobiera do ustalonych parametrów (wszystkie są w configurator.py)
-        print(config.analysis['sampling_frequency'])
\ No newline at end of file
+        pass
+
+def normalziation(self,audio):
```

Comment:
Cała metoda znajduje się poza klasą. Żeby to naprawić musisz dodać tabulator przed całą metodą.
Dodatkowo brakuje spacji przed audio i nazwę zmieniłbym na normalize